### PR TITLE
✨👽️ (feature + api) : Update child feature and location address-autocomplete

### PR DESCRIPTION
**Pull request by d-azmanov**
- New page for Updating a child
- Added OpenStreetMaps API calls to auto-complete addresses when searching for new location for adding an activity
- Added OpenStreetMaps API calls to auto-complete addresses when searching for new location for editing an activity

### CODE REVIEW

**Review by priyo5 — DISMISSED** [review]

File: libs/edit-child/feature/src/lib/edit-child-feature.module.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { CommonModule } from '@angular/common';
+import { EditChildComponent } from './edit-child.component';
+import { EditChildRoutingModule } from './edit-child-routing.module';
+import { FormsModule } from '@angular/forms';
+import { NavbarModule } from '@the-au-pair/shared/components/navbar';
+import { IonicModule } from '@ionic/angular';
+import { API } from '../../../../shared/api/api.service';
+
+@NgModule({
+  imports: [CommonModule,
```
Comment (ricecakesreece) [inline libs/edit-child/feature/src/lib/edit-child-feature.module.ts:11]: For neatness just have CommonModule on a new line

File: libs/edit-child/feature/src/lib/edit-child.component.scss [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    align-items: center;
+    flex-direction: column;
+    text-align: right;
+}
+
+// Button styling
+.childrenDetails input{
+    width: 411px;
+    height: 62px;
+    background: #F9F9F9;
```
Comment (ricecakesreece) [inline libs/edit-child/feature/src/lib/edit-child.component.scss:53]: In future, this colour has now been added to themes and is called --ion-color-light-white

File: libs/edit-child/feature/src/lib/edit-child.component.scss [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+    border-style: none;
+    border-radius: 10px;
+    padding-top: 10px;
+    padding-left: 20px;
+    color: black;
+}
+
+label{
+    position:relative;
+    color: #797979;
```
Comment (ricecakesreece) [inline libs/edit-child/feature/src/lib/edit-child.component.scss:75]: For future reference this colour is now going to be called --ion-color-grey

File: libs/edit-child/feature/src/lib/edit-child.component.scss [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+    margin: 0;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 56px;
+    text-transform: capitalize;
+    position: relative;
+    text-align: center;
+    margin-bottom: -40px;
+    color: #3C3C3C;
```
Comment (ricecakesreece) [inline libs/edit-child/feature/src/lib/edit-child.component.scss:26]: --ion-color-dark

File: libs/edit-child/feature/src/lib/edit-child.component.scss (hunk 2 above)
Comment (d-azmanov) [reply libs/edit-child/feature/src/lib/edit-child.component.scss:53]: It hasn't been added yet, but when it is, I will update this

**Review by PxttytheThxtty — APPROVED** [review]

**Review by priyo5 — APPROVED** [review]

**Review by ricecakesreece — APPROVED** [review]

**Pull request merged**